installers/linux/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:16.04
```

Comment:
Could you switch this to using `gcr.io/gcp-runtimes/ubuntu_16_0_4`? That way we don't have rely on dockerhub being to release kvm.

Reply:
Ok